webview-ui/src/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+
+ReactDOM.render(
```

Comment:
We can use the newer React functions like `createRoot` https://reactjs.org/docs/rendering-elements.html#rendering-an-element-into-the-dom